### ISSUE TITLE
Facebook Strategy - Access Token

### ISSUE THREAD

**Issue by ianlivingstone** [ciaranj/connect-auth#65]
How can I retrieve the access token via the Facebook Strategy, is it not exposed somehow?

**Comment by ciaranj** [comment]
Yup. It is stored in the session as 'access_token' however this is actually an error and it should be accessible via the getAuthDetails helper method. For now that session variable should work for you?

**Comment by ianlivingstone** [comment]
Yes that works for now
